staff/sean-daryanani/cv-styled-sass/index.html

```diff
@@ -0,0 +1,140 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sean Curriculum</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+        <header class="header">
+            <div class="image-container">
+                <h1 class="image-container__title">Sean Daryanani <span class="image-container__span">Fullstack Web Developer</span></h1>
+                <img src="https://www.yourbusinessfreedom.com.au/wp-content/uploads/2017/02/unknown-profile-1.jpg"
+                    width="150px" alt="" class="img-container__img">
+            </div>
+            <address class="address">
+                <p>Phone number: <span class="address__span">XXXXXXXXXXX</span></p>
+                <p>Ciudad: <br> <span class="address__span">Barcelona</span></p>
+                <p>Email: <a href="mailto:seand52@gmail.com?Subject=Job Opportunity" target="_top">seand52@gmail.com</a></p>
+            </address>
+
+        </header>
+        <main>
+            <section class="goals">
+                <h2 class="goals__title">Career Goals</h2>
+                <p class="goals__description">I am an aspiring developer currently undertaking a web development bootcamp at Skylab Academy. I
+                    believe
+                    my
+                    hard-working attitude will allow me to thrive in this field.</p>
+            </section>
+
+            <section class="skills">
+                <h2 class="skills__title">Skills</h2>
+                <div class="skills-container">
+                    <div class="skills-container__programming">
+                        <p>Programming Languages</p>
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                            <li>Python</li>
+                            <li>R</li>
+                        </ul>
+                    </div>
+                    <div class="skills-container__data">
+                        <p>Data Analysis Software</p>
+                        <ul>
+                            <li>Excel</li>
+                            <li>SPSS</li>
+                            <li>Q Research</li>
+                        </ul>
+                    </div>
+                </div>
+
+
+            </section>
+
+            <section class="projects">
+                <h2 class="projects__title">Projects</h2>
+                <ul class="projects__list">
+                    <li><b>Pasapalabra Game</b> - This game is was made using HTML, CSS, and Javascript</li>
+                    <li><b>Calculator</b> - Web-based calculator that uses HTML, CSS, and Javascript</li>
+                </ul>
+            </section>
+
+            <section class="work">
+                <h2 class="work__title">Work Experience</h2>
+               
+                    <section class="work__description">
```

Comment:
may work__description be another block-element by itself? then in that case you would have two block elements:
- work
- work-description
each one with its sub-classes (sub-elements and modifiers)
and may work-description be nest-able inside work